.github/dependabot.yml

```diff
@@ -3,13 +3,13 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: monthly
-    time: '03:00'
-  open-pull-requests-limit: 5
+    interval: weekly
   ignore:
   - dependency-name: electron
-    versions:
-    - "> 5.0.13"
+  - dependency-name: spectron
+  - dependency-name: typescript
+  - dependency-name: sass-loader
+    versions: ["11.x"]
```

Comment:
V11 breaking the app, because of incorrect dependencies with Vuetify or something.